src/util/allocate_objects.cpp

```diff
@@ -271,3 +271,15 @@ void allocate_objectst::mark_created_symbols_as_input(code_blockt &init_code)
     init_code.add(std::move(input_code));
   }
 }
+
+/// Create code allocating object of size `size` and assigning it to `lhs`
```

Comment:
"an object"

Reply:
Done in a new "Tidy up documentation" commit. :shell: